maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .book-info-container {
+    flex-grow: 0;
+    margin-right: 0;
+  }
+
+  .book-title-container {
+    margin-bottom: 0;
```

Comment:
Be careful with these things. You are separating the title from the content through the title's margin bottom and it should the parent `book-title-container` the one doing that. 

![image](https://user-images.githubusercontent.com/36488701/59545202-41902280-8ee9-11e9-82c8-14d9b673c304.png)